ethcore/src/builtin.rs

```diff
@@ -101,18 +101,17 @@ impl Pricer for ModexpPricer {
 		let exp_len = read_len();
```

Comment:
It won't let me add comments on the actual line, but on line 92 above here: Why do we zero-extend the input and `read_exact` instead of just using `read`? The buffer's already filled with zeroes.